File: Semestru 4/MPP/lab/java/lab3/Services/src/main/java/org/example/services/IServiceChild.java
package org.example.services;

import org.example.domain.Child;

public interface IServiceChild {
    Iterable<Child> GetAll() ;
    Child GetById(int id);
    Child AddChild (String name, String cnp) ;
    Child UpdateChild (Child child) ;
    Iterable<Child> GetByAge (int age) ;
}

File: Semestru 4/MPP/lab/.net/lab3/lab3/Services/services/IServiceChild.cs
using Model.app.domain;


namespace Services.services
{
	public interface IServiceChild
	{
		IEnumerable<Child> GetAll();
		Child GetById(int id);
		Child Add(string name, string cnp);
		Child Update(Child child);
		IEnumerable<Child> GetByAge(int age);
	}
}
